.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "build"]
 	path = build
-	url = https://github.com/upbound/build
+	url = https://github.com/soorena776/build
```

Comment:
this is temporary, and should revert once 
https://github.com/upbound/build/pull/74
is approved